census/views/base.html

```diff
@@ -1,4 +1,4 @@
-{% set godi = "false" %}
+{% set godi = site.settings.is_godi or false %}
```

Comment:
@brew `godi` is too specific ("global"), I'd say the flag can be `is_index` ("Survey" or "Index").